lib/jekyll/rp_logs/rp_logline.rb

```diff
@@ -145,7 +145,13 @@ def mergeable_with?(next_line)
       end
 
       def merge!(next_line)
-        @contents += "#{space_between_lines}#{next_line.contents}"
+        if /\.\.\.\w*$/.match(@contents) && /^\w*\.\.\./.match(next_line.contents)
```

Comment:
But those don't catch trailing spaces, they would match, like,
`Lorem ipsum...abc` and `def...dolor sit amet` and merge it into `Lorem ipsum dolor sit amet`.
Is there an example file where there was an issue that this solved? I'd like to make a test out of it if so.